pandas/tests/indexes/test_multi.py

```diff
@@ -3291,3 +3291,37 @@ def test_duplicate_multiindex_labels(self):
         with pytest.raises(ValueError):
             ind.set_levels([['A', 'B', 'A', 'A', 'B'], [2, 1, 3, -2, 5]],
                            inplace=True)
+
+    testdata = [
+        (pd.MultiIndex.from_product([[0, 1], [1, 0]]), pd.Series(range(4)), 4),
```

Comment:
don't create a variable here, just put it directly in the parametrize (see how to format it from other tests)